static/js/components/FollowupRequestForm.jsx

```diff
@@ -27,21 +32,32 @@ const FollowupRequestForm = ({
   const { allocationList } = useSelector((state) => state.allocations);
   const allGroups = useSelector((state) => state.groups.all);
   const [selectedAllocationId, setSelectedAllocationId] = useState(null);
+  const [selectedGroupIds, setSelectedGroupIds] = useState([]);
 
   useEffect(() => {
     // Initialize the select
     setSelectedAllocationId(allocationList[0]?.id);
-  }, [setSelectedAllocationId, allocationList]);
+    setSelectedGroupIds([allocationList[0]?.group_id]);
+  }, [setSelectedAllocationId, allocationList, setSelectedGroupIds]);
 
   // need to check both of these conditions as selectedAllocationId is
   // initialized to be null and useEffect is not called on the first
   // render to update it, so it can be null even if allocationList is not
   // empty.
-  if (allocationList.length === 0 || !selectedAllocationId) {
+  if (
+    allocationList.length === 0 ||
```

Comment:
`allocationList?.length`?

Reply:
No need, `allocationList` is initialized in redux to be `[]`.